WFP-VAM/prism-app
# Use correct wcsconfig parameter for rasters without time support COUNTRY=mozambique

Closes #600 

This PR replaces the parameter to remove the time parameter used for the geotiff_url within a /stats request to the API..

**How to test:**
- Select Tropical Storms layer and press the **Exposure analysis** button. Verify within the network tab that the `geotiff_url` does not have the time parameter.
- Run a normal analysis and verify that the parameter appears.
